apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/FissClaimTransformerV2.java

```diff
@@ -231,83 +136,44 @@ private static Resource getContainedProvider(RdaFissClaim claimGroup) {
-      organization
-          .getIdentifier()
-          .add(
-              new Identifier()
-                  .setType(
-                      new CodeableConcept(
-                          new Coding(
-                              C4BBIdentifierType.NPI.getSystem(),
-                              C4BBIdentifierType.NPI.toCode(),
-                              C4BBIdentifierType.NPI.getDisplay())))
-                  .setSystem(TransformerConstants.CODING_NPI_US)
-                  .setValue(claimGroup.getNpiNumber()));
-    }
-
+    addFedTaxNumberIdentifier(
+        organization, BBCodingSystems.FISS.TAX_NUM, claimGroup.getFedTaxNumber());
+    addNpiIdentifier(organization, claimGroup.getNpiNumber());
     organization.setId("provider-org");
 
     return organization;
   }
 
+  /**
+   * Parses data from the {@link RdaFissClaim} to create a facility {@link Reference}.
+   *
+   * @param claimGroup The {@link RdaFissClaim} object to parse data from.
+   * @return The facility {@link Reference} object created from the parsed data.
+   */
   private static Reference getFacility(RdaFissClaim claimGroup) {
-    if (claimGroup.getLobCd() != null || claimGroup.getNpiNumber() != null) {
-      Reference reference = new Reference();
+    Reference reference;
 
-      if (claimGroup.getNpiNumber() != null) {
-        reference.setIdentifier(
-            new Identifier()
-                .setType(
-                    new CodeableConcept(
-                        new Coding(
-                            C4BBIdentifierType.NPI.getSystem(),
-                            C4BBIdentifierType.NPI.toCode(),
-                            C4BBIdentifierType.NPI.getDisplay())))
-                .setSystem(TransformerConstants.CODING_NPI_US)
-                .setValue(claimGroup.getNpiNumber()));
-      }
-
-      if (claimGroup.getLobCd() != null) {
-        reference.setExtension(
-            List.of(
-                new Extension(BBCodingSystems.CLM_FACILITY_TYPE_CODE)
-                    .setValue(
-                        new Coding(
-                            BBCodingSystems.CLM_FACILITY_TYPE_CODE, claimGroup.getLobCd(), null))));
-      }
-
-      return reference;
+    if (Strings.isNotBlank(claimGroup.getLobCd())) {
+      reference = new Reference();
+      addExtension(reference.getExtension(), BBCodingSystems.FISS.LOB_CD, claimGroup.getLobCd());
+    } else {
+      reference = null;
     }
 
-    return null;
+    return reference;
   }
 
+  /**
+   * Parses data from the {@link RdaFissClaim} to create a {@link Claim.DiagnosisComponent} list.
+   *
+   * @param claimGroup The {@link RdaFissClaim} object to parse data from.
+   * @return The {@link Claim.DiagnosisComponent} object list created from the parsed data.
+   */
   private static List<Claim.DiagnosisComponent> getDiagnosis(
       RdaFissClaim claimGroup, boolean isIcd9) {
     final String icdSystem = isIcd9 ? IcdCode.CODING_SYSTEM_ICD_9 : IcdCode.CODING_SYSTEM_ICD_10;
```

Comment:
@KyRobbins based on the latest from Bill Harty, we'll want to use `CODING_SYSTEM_ICD_10_CM`.  (CODING_SYSTEM_ICD_9 is correct though)